src/legacy/core_plugins/kibana/public/visualize/listing/visualize_listing_table.js

```diff
@@ -256,94 +390,295 @@ class VisualizeListingTableUi extends Component {
   };
 
   deleteSelectedItems = () => {
-    this.props.deleteSelectedItems(this.state.selectedRowIds)
+    this.props.deleteSelectedItems(this.state.selectedIds)
       .then(() => this.fetchItems(this.state.filter))
       .catch(() => {})
       .then(() => this.deselectAll())
-      .then(() => this.closeModal());
+      .then(() => this.closeDeleteModal());
```

Comment:
While the modal waits for the items to get deleted, can you disable the modal buttons and change the "Delete" button text to "Deleting"? This way the user knows that their action has been triggered because, right now, it does nothing visual in response to the user clicking "Delete".

Reply:
Changed text and disabled clicking on the button again, but can't disable the button visually, because it's not supported on the component.